mujoco_py/mjsim.pyx

```diff
@@ -1,5 +1,8 @@
 from xml.dom import minidom
 from mujoco_py.utils import remove_empty_lines
+from threading import Lock
+
+render_lock = Lock()
```

Comment:
this name is too generic to be in the global namespace. Let's call it `_MjSim_render_lock`